generally recommend using remote bindings over `--remote` (#25256)
cloudflare/cloudflare-docs: src/content/docs/browser-rendering/workers-bindings/screenshots.mdx

## 6. Test

> Run `npx wrangler dev` to test your Worker locally, you can also use [remote bindings](/workers/development-testing/#remote-bindings) to use the production browser rendering functionality, or run [`npx wrangler dev --remote`](/workers/wrangler/commands/#dev) to test your Worker remotely before deploying to Cloudflare's global network.

Reviewer: ```suggestion
Run `npx wrangler dev` to test your Worker locally. You can also use [remote bindings](/workers/development-testing/#remote-bindings) to test against a remote browser.
```
